### ISSUE TITLE
Status check through cloudflare tunnel or Tailscale

### ISSUE THREAD

**Issue by goake** [Lissy93/dashy#1673]
### Question

I'm really struggling to get this Status Check to work. Using Potainer as an example.

I have an Oracle Cloud VM set up, Potainer running in a container. Only ports on the VM that are open are 80 and 443
nginx in another container is configured to proxy https://port.mydomain.com to http://portainer:9000
I also have my domain managed by CloudFlare, which obfiscates the VM's public IP 
The VM is also part of a Tailscale network, and has the name vm1

If I set the Service URL to https://port.mydomain.com then the Status check is always healthy, even if i shut the portainer container down. I assume that this is Cloudflare returning a healthy status, wheather the Portainer container is running or not.

If I set the Service URL to http://vm1:9000 (using the Tailscale VM name) then although the link works, and successfully opens up Portainer, the Status Check fails with error ECONNREFUSED

If I open up port 9000, and set the Service URL to http://publicIP:9000 then both the link and the status check works perfectly

Surely there must be a way for me to achieve this without opening up port 9000 on the vm?

### Category

Setup and Deployment

### Please tick the boxes

- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy (check the first two digits of the version number)
- [X] You've checked that this [question hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by goake** [comment]
I've got this working. I dont know if this is correct, or just a work around. This is what I've done:

Service URL is set to the public FQDN ie https://port.mydomain.com which is proxied by CloudFlare to the vm and then nginx to the portainer container port 9000

I've used Status Check URL in either:

- if dashy is on the same host as portainer, then used the container name ie http://portainer:9000
- if dashy is on another host, in the same Tailscale network, used the Taliscale IP address of the portainer host ie http://IPaddress:9000

All seems to work. It links to the application, and the status indicator reflects correctly if the container is running or not
